partials/footer-mail-verification.mjml

```diff
@@ -0,0 +1,21 @@
+<mj-section css-class="section" padding-bottom="16px" padding-top="0px">
+  <mj-column>
+    <mj-text
+      css-class="footer-text"
+      font-size="14px"
+      line-height="20px"
+      color="#17324D"
+    >
+      <p style="margin-top: 0px; font-size: 14px">
+        Ricevi questa comunicazione perch&eacute; hai inserito questo indirizzo e-mail come recapito di cortesia di SEND. Per modificare i recapiti o aggiungerne altri,
+        <a
+          title="Continua su Piattaforma Notifiche"
+          class="link"
+          href="https://portale.dev.pn.pagopa.it/recapiti"
```

Comment:
@SarahDonvito the link here is wrong, should be "https://cittadini.notifichedigitali.it/recapiti"